tools/ci/flink-ci-tools/src/main/java/org/apache/flink/tools/ci/licensecheck/NoticeFileChecker.java

```diff
@@ -62,6 +62,10 @@ public class NoticeFileChecker {
     private static final Pattern SHADE_INCLUDE_MODULE_PATTERN =
             Pattern.compile(".*Including ([^:]+):([^:]+):jar:([^ ]+) in the shaded jar");
 
+    private static final Pattern DEPLOY_MODULE_PATTERN =
```

Comment:
This would be easier to understand if we added a comment showing an example of a string this pattern is applied to.